### ISSUE TITLE
Need more version requirements

### ISSUE THREAD

**Issue by CharlesSS07** [computationalpathologygroup/ASAP#204]
I am building on centOS7, and I keep not having the right packages even though there are no explicit package requirements for ASAP. It would save quite a bit of time for people who are intending to build this package if there were more explicit version requirements.

If you could say what versions of packages, exactly, that worked for you, that would be amazing.

Thanks,
Charles.

**Comment by HansPinckaers** [comment]
I agree, that would be useful. For now maybe you can take a look at the logs on appveyor? 

For example: https://ci.appveyor.com/project/GeertLitjens/asap/builds/39525557/job/6fldwl4enblbn61g
